examples/sre/v3/path.sh

```diff
@@ -0,0 +1,8 @@
+export PATH=$PWD:$PATH
+
+# NOTE(kan-bayashi): Use UTF-8 in Python to avoid UnicodeDecodeError when LC_ALL=C
+export PYTHONIOENCODING=UTF-8
+export PYTHONPATH=../../../:$PYTHONPATH
+
+export PATH=$PATH:/mnt/matylda6/rohdin/software/kaldi_20200214/tools/sph2pipe_v2.5/
```

Comment:
Since installing Kaldi is a little complex, we only use some useful shell/perl/python scripts in WeSpeaker rather than installing the whole Kaldi. 
Thus I recommend two methods here:
1. Download [sph2pipe_v2.5.tar.gz](https://www.openslr.org/3/) and decompress it into a ``external_tools`` dir;
2. Use some other tools to convert sph into wav, i.e., ffmpeg.

Reply:
Sure, I'll fix it. I'm on a trip this week so probably next week.

Reply:
@gulamungon Hi Johan, can you fix the lint errors, .etc, then we can proceed the merging process